optaplanner-examples/data/machinereassignment/unsolved/model_a1_1.json

```diff
@@ -0,0 +1,2532 @@
+{
+  "id" : 0,
+  "globalPenaltyInfo" : {
+    "id" : 0,
+    "processMoveCostWeight" : 1,
+    "serviceMoveCostWeight" : 10,
+    "machineMoveCostWeight" : 100
+  },
+  "resourceList" : [ {
+    "id" : 0,
+    "index" : 0,
+    "transientlyConsumed" : false,
+    "loadCostWeight" : 10
+  }, {
+    "id" : 1,
+    "index" : 1,
+    "transientlyConsumed" : false,
+    "loadCostWeight" : 10
+  } ],
+  "neighborhoodList" : [ {
+    "id" : 0
+  } ],
+  "locationList" : [ {
```

Comment:
Isn't it strange that this list is not ordered by ID? This suggest to me that some collection somewhere isn't in a consistent order.

Reply:
Actually, the collection is in a consistent order:
https://github.com/kiegroup/optaplanner/blob/main/optaplanner-examples/data/machinereassignment/unsolved/model_a1_1.xml#L28
Maybe it wasn't in the past when the dataset was created.